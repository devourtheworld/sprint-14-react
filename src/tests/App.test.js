"use strict"

import App from './../App';
import TestRenderer from 'react-test-renderer';
import dataService from './../App';
import state from './../App';


describe('Rendering test', () => {
  it('renders correctly', () => {
    const wrapper = TestRenderer.create(<App/>);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});

test('test', () => {
  state = {};
  expect(state).toBe(`https://api.github.com/users/zbys1av`);
});
