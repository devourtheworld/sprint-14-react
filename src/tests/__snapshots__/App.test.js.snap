// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering test renders correctly 1`] = `
<div>
  <h3>
    GitHub User Info
  </h3>
  <ul />
</div>
`;
